crates/uv-client/src/error.rs

```diff
@@ -236,6 +236,9 @@ pub enum ErrorKind {
     #[error("Invalid `Content-Type` header for {0}")]
     InvalidContentTypeHeader(Url, #[source] http::header::ToStrError),
 
+    #[error("Invalid status code: {0}")]
+    InvalidStatusCode(String),
```

Comment:
Can we retain the type? In other words, should this be `#[from] IndexSourceError`? Why `String`?

Reply:
This is going to go away now that I’m going to wrap StatusCode

Reply:
Removed with refactor